pkg/kubectl/cmd/version.go

```diff
@@ -56,7 +57,7 @@ func NewCmdVersion(f cmdutil.Factory, out io.Writer) *cobra.Command {
 	}
 	cmd.Flags().BoolP("client", "c", false, "Client version only (no server required).")
 	cmd.Flags().BoolP("short", "", false, "Print just the version number.")
-	cmd.Flags().String("output", "", "output format, options available are yaml and json")
+	cmd.Flags().StringP("output", "o", "", "output format, options available are yaml and json")
 	cmd.Flags().MarkShorthandDeprecated("client", "please use --client instead.")
```

Comment:
When we run `kubectl version -c`, the output will be `Flag shorthand -c has been deprecated, please use --client instead.` So we should leave it as is.